src/Entities/ui/AsideItemWrapper/AsideItemWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, ReactNode } from 'react'
+
+interface AsideItemWrapperProps {
+  children: ReactNode
+  onClick?: () => void
+  isChosen: boolean
+  label: string
+}
+
+const AsideItemWrapper: FC<AsideItemWrapperProps> = ({
+  children,
+  label,
+  isChosen,
+  onClick,
+}) => {
+  const handleClick = () => {
+    if (onClick) onClick()
```

Comment:
Обычно в таких случаях оборачивают в скобки, чуть лучше читаемость. Или можно просто второй вариант. Но тут тоже кому как нравится
```suggestion
    if (onClick) {
      onClick()
    }
```
```suggestion
    onClick?.()
```